pmg/bills.py

```diff
@@ -111,9 +111,61 @@ def bill_history(bill):
         info.update(location)
         history.append(info)
 
+    history = hansard_linking(history)
     return history
 
 
+def match_title(event_title):
+    """
+    Match bill title against the following possible titles
+    """
+    bill_titles = [
+        "Bill passed by National Assembly",
+        "Bill passed by both Houses",
+        "Bill revived on this date",
+        "The NCOP rescinded",
+        "Bill remitted",
+    ]
+    for title in bill_titles:
+        if title in event_title:
+            return True
+    return False
+
+
+def match_dates(hansard_date, event_date):
+    hansard_iso_date = iso8601.parse_date(hansard_date)
+    event_iso_date = iso8601.parse_date(event_date)
+    if hansard_iso_date.date() == event_iso_date.date():
+        return True
+    return False
+
+
+def hansard_linking(bill_history):
+    """
+    We need to link certain bill events to hansards
```

Comment:
I think it's worth documenting:

- does this modify one or more events? Does it modify al events that meet the criteria?
- Does this do anything special if there are more than one appropriate hansards or does it just pick _some_ matching hansard without warning?